site/src/components/hero.js

```diff
@@ -9,15 +9,15 @@ const Hero = () => (
     <div className="grid-container">
       <div className="usa-hero__callout">
         <h2 className="usa-hero__heading">
-          <span className="usa-hero__heading--alt">Hero callout:</span>Bring
-          attention to a project priority
+          <span className="usa-hero__heading--alt">Find Testing:</span>
+          Nearby test centers for COVID-19
         </h2>
         <p>
-          Support the callout with some short explanatory text. You don’t need
-          more than a couple of sentences.
+          Think you may have the virus or are concerned about recent exposure?
+          Fill out a couple questions to find the best course of action.
         </p>
         <a className="usa-button" href="/">
```

Comment:
to point to the same page in the navbar:
```suggestion
        <a className="usa-button" href="/should-i-test">
```